Loop/Managers/Alerts/AlertStore.swift

```diff
@@ -56,6 +60,7 @@ extension AlertStore {
             do {
                 try self.managedObjectContext.save()
                 self.log.default("Recorded alert: %{public}@", alert.identifier.value)
+                self.purgeExpired()
```

Comment:
[nit] This, of course, means that if the user doesn't get an alert for a while, that the purging would not happen, possibly well beyond the 90 days.  I presume, in practice, this is not an issue?  (Not blocking, just wondering what the pattern normally is) Otherwise, we might want to `public`-ize `purgeExpired` to be run as part of a regular CGM update or (or some such).  Just a thought...

Reply:
Yes, that is correct. I don't see that as an issue. If nothing has changed, then I see no reason to purge the cache. There will be, in another ticket, code that will purge all caches to shorter and shorter expirations upon device low storage conditions.

Besides, the total storage space for the alert store, even given a fairly large number of alerts/day, is zilch compared to the rest of the stores. That is, 90 days worth of simulated data for all of the other stores is 2GB, while this is maybe a few MB. And, if they really are getting lots of alerts every day, then the store will be purged frequently as well.